cob_bringup/components/cam3d_sick_visionary.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+	<arg name="name"/>
+	<arg name="sim" default="false"/>
+
+	<include file="$(find cob_bringup)/drivers/sick_visionary_t.launch">
```

Comment:
add an `unless="$(arg sim)"` for the driver as in https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/components/cam3d_openni2.launch#L13